application/org.openjdk.jmc.flightrecorder.ui/src/main/resources/org/openjdk/jmc/flightrecorder/ui/messages/internal/messages.properties

```diff
@@ -522,6 +522,10 @@ ThreadsPage_TABLE_POPUP_TITLE=Thread Details
 TlabPage_PAGE_NAME=TLAB Allocations
 TlabPage_THREADS_TAB_NAME=By Threads
 TlabPage_METHODS_TAB_NAME=By Top Methods
+TlabPage_CLASS_TAB_NAME=By Class
+TlabPage_SUMMARY_TAB_NAME=General
```

Comment:
Minor nit, but I think "Summary" better describes the page than "General", and also follows the naming convention used in the code.

Reply:
Agreed. Even I thought of Summary, but then since JMC 5.5 was using General so thought of replacing that literal to make it look similar to customer. They are strictly comparing both JMC 5.5 and JMC 8. But summary sounds better to I did this change.